features/directconnect/directconnect.feature

```diff
@@ -4,12 +4,9 @@ Feature: AWS Direct Connect
 
   I want to use AWS Direct Connect
 
-  Scenario: Managing connections
-    Given I create a Direct Connect connection with name prefix "aws-sdk-js"
-    Then I should get a Direct Connect connection ID
-    And I describe the connection
-    Then the bandwidth should match the connection bandwidth
-    And I delete the Direct Connect connection
+  Scenario: describe connections
+    When I describe the connection
+    Then I should get response of type "Array"
```

Comment:
We have a standard way of testing list/describe operations in most of our feature tests that look like this:
https://github.com/aws/aws-sdk-js/blob/v2.135.0/features/acm/acm.feature#L7-L10

If you follow this patten, you don't have to create your own step definitions, since cucumber will use the ones defined here:
https://github.com/aws/aws-sdk-js/blob/v2.135.0/features/extra/hooks.js#L56